packages/cli/lib/cli.ts

```diff
@@ -313,43 +314,38 @@ export const tscWatch = async (debug = false) => {
     watcher.on('change', (filePath: string) => {
         if (filePath === nangoConfigFile) {
             // config file changed, re-compile each ts file
-            const integrationFiles = glob.sync(`./*.ts`);
+            const integrationFiles = listFiles({});
```

Comment:
weird interface to pass an empty object to a function

Reply:
yep true